pkg/controller/constraint/stats_reporter.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 const (
-	totalConstraintsName = "total_constraints"
+	totalConstraintsName = "constraints"
```

Comment:
Description for constraintsTotalM should be updated

Reply:
Changed to: "Current number of known constraints"